.travis.yml

```diff
@@ -23,15 +23,15 @@ services:
 before_install:
   - echo "deb http://archive.ubuntu.com/ubuntu trusty-backports main restricted universe multiverse" | sudo tee -a /etc/apt/sources.list
   - sudo apt-get -qq update
-  - sudo apt-get install -y libseccomp-dev/trusty-backports genisoimage
+  - sudo apt-get install -y libseccomp-dev/trusty-backports genisoimage jq
```

Comment:
Assume that `jq` is just for the test, but can we add it into installation of the README stuff - or just make the README use `make before-install`.